hotsos/cli.py

```diff
@@ -369,4 +382,11 @@ def cli(data_root, version, defs_path, templates_path, all_logs, quiet,
 
 
 if __name__ == '__main__':
+    if is_snap():
+        if get_os_id() != 'ubuntu':
+            print("This snap has only been verified to run on Ubuntu")
+            sys.exit(1)
+        if get_os_codename() not in ["focal"]:
+            print("This snap has only been verified to run on Focal")
+            sys.exit(2)
```

Comment:
I thought of having --force override this, but probably best to just exit, or use pip :-)

Reply:
we need to allow it work on >= focal. we test py3.8, py3.10 and py3.11 currently so it should be good for anything above and including focal.

Reply:
Fixed